### ISSUE TITLE
"SyntaxError: Unexpected token -" In Browser

### ISSUE THREAD

**Issue by ElMatella** [commenthol/safer-eval#3]
Hi, I am trying to use your package (thank you for creating it by the way :) ). But it seems that it does not work on the browser:

```
import saferEval from 'safer-eval'
console.log('the result' + saferEval('1 + 1'))
```
This code results in a "SyntaxError: Unexpected token -" in the browser. The same code in nodeJS works...

EDIT: Ho, sorry, it seems to be fixed by #1. Are you planning to merge it anytime soon and make a release of it?

**Comment by commenthol** [comment]
Unfortunately your statement above that "1+1" is broken is not reproducible here. Please check https://github.com/commenthol/safer-eval-demo 
The issue has to deal with your project setup.

**Comment by Nishchit14** [comment]
@commenthol you have used `safe-eval` not `safer-eval`

https://github.com/commenthol/safer-eval-demo/blob/master/index.js#L2

**Comment by commenthol** [comment]
@Nishchit14 thanks. Is fixed now. Same result 1 + 1 = 2.

**Comment by ElMatella** [comment]
@commenthol see https://github.com/ElMatella/safer-eval-demo to reproduce. 

Problem happen when some variable is defined with a dash on the global scope:

```
window['test-with-something'] = 5
```

It breaks with error message:

```
Uncaught SyntaxError: Unexpected token -
```

**Comment by commenthol** [comment]
The issue is fixed with v1.3.1. It took me some time to understand the case.  
The line `window['test-with-something'] = 5` was crucial to me.
